hummingbot/connector/exchange/wazirx/wazirx_exchange.py

```diff
@@ -83,6 +82,8 @@ def __init__(self,
         super().__init__()
         self._trading_required = trading_required
         self._trading_pairs = trading_pairs
+        self._account_available_balances = {}
+        self._account_balances = {}

```

Comment:
These variables are defined in the superclass. The should not be redefined here.

Reply:
Thanks, updated.